### PR TITLE
[WIP] Dev ltclm windatlas fix

### PR DESCRIPTION
**Pull request by ltclm**
@AFoletti  Could you please check if everything is now as discussed yesterday.
[Testlink](https://mf-geoadmin3.dev.bgdi.ch/dev_ltclm_windatlas_fix/?lang=de&topic=ech&bgLayer=ch.swisstopo.pixelkarte-farbe&layers=ch.bfe.windenergie-geschwindigkeit_h50,ch.bfe.windenergie-geschwindigkeit_h75,ch.bfe.windenergie-geschwindigkeit_h100,ch.bfe.windenergie-geschwindigkeit_h125,ch.bfe.windenergie-geschwindigkeit_h150&layers_visibility=true,false,false,false,false)

### CODE REVIEW

**Comment by AFoletti** [comment]
Looks good. Still one point open: could you please round to 1 place after comma
- Average wind speed (both tooltip and extended)
- A and k attributes (extended tooltip)
- Frequency (extended tooltip, second table)

**Comment by ltclm** [comment]
@AFoletti it's done

**Comment by gjn** [comment]
Legend images are not transparent. Correct?

File: chsdi/templates/htmlpopup/windatlas50.mako [hunk 1]
```diff
@@ -230,92 +232,92 @@ td.inner-table {
         </tr>
         <tr>
           <td>total</td>
-          <td>${round(freq_total, 3)} %</td>
-          <td>${round(props['v_mean'], 3)}</td>
-          <td>${round(props['wei_a'], 3)}</td>
-          <td>${round(props['wei_k'], 3)}</td>
+          <td id="align-right">${round(freq_total, 1)} %</td>
```
Comment (gjn) [inline chsdi/templates/htmlpopup/windatlas50.mako]: We should use `class` not `id`. Per standard, `id` should be unique in a html document.

Comment (ltclm) [inline chsdi/templates/htmlpopup/windatlas50.mako]: class did not work, its overriden, that's why i have added a class.

Comment (gjn) [inline chsdi/templates/htmlpopup/windatlas50.mako]: hmm. Overridden how? Maybe use a different name?

Comment (gjn) [inline chsdi/templates/htmlpopup/windatlas50.mako]: Or you can use parent class `td.kernkraftwerke-extended > .myrighalign`. I'm sure there's a way.

**Comment by AFoletti** [comment]
LGTM. Ready for merge if OK for @gjn

**Comment by AFoletti** [comment]
About the legends, those were delivered by the datenherr. They are not transparent but it's not a critical issue in my opinion.

**Comment by gjn** [comment]
Great!

**Pull request merged**